mintapi/api.py

```diff
@@ -846,6 +851,18 @@ def get_invests_json(self):
         else:
             logger.error("FAIL2")
 
+    def get_investment_data_new(self):
```

Comment:
Can we leave off the suffix "new"? I generally dislike time linked function names.